### ISSUE TITLE
DS4 Windows Issue (Unable to Install Drivers) {WEBSITE is OFFLINE}

### ISSUE THREAD

**Issue by xdantey** [Jays2Kings/DS4Windows#254]
Hello,

I've decided to reinstall my DS4 Drivers because I had an issue with my PS4 Gamepad, it suddenly stopped getting any signal from DS4 Windows and acted as a simple PS4 Gamepad.
Upon reinstalling, all DS4Windows is doing is "Opening Installer" for several minutes and fail to do so, I've tried to go on the official website to grab another set of files in case those became corrupt and to my surprise, the website has gone offline.

Apparently the bandwidth limit had been exceeded and I'm pretty sure the installer gets files from said website, it opens the installer and downloads the required files to install DS4 Drivers.

Could you help me out solve this problem? Do you have a copy of your DS4 Windows for offline installation usage?

**Comment by ghost** [comment]
@xdantey The site appears to be up at this time. If you are referring to "Step 1: Install the DS4 Driver", then you are correct in regards to the installer retrieving a file from the site (http://ds4windows.com/Files/Virtual Bus Driver.zip). I would suggest downloading that file if there have been intermittent site outages. Simply running and installing ScpDriver should address the first step.

**Comment by sweetkid169** [comment]
so im getting the 

Bandwidth Limit Exceeded

The server is temporarily unable to service your request due to the site owner reaching his/her bandwidth limit. Please try again later.

error when i try to just pull up the standard site... does anyone have the vbus file downloaded? if so can your share it? cuz the site is most def not functioning ATM.

**Comment by Jimmy-Z** [comment]
I'm also hitting this, why is that not included in the release package>

**Comment by sweetkid169** [comment]
i found a workaround
https://ds4-to-xinput-wrapper.en.softonic.com
this thing has the virtual bus file with it! grab and install! ds4 is now working and detects my controller!!! woot!

**Comment by Gumija** [comment]
You can get the zip from WayBack Machine. (https://web.archive.org/web/20170311022846/http://ds4windows.com/Files/Virtual%20Bus%20Driver.zip)
But what do you do with it after? Just run ScpDriver.exe?

**Comment by Gumija** [comment]
Yes, just run ScpDriver.exe and after that start DS4Windows and the Install window won't pop up (at least on win10)

**Comment by sweetkid169** [comment]
yup. and i couldn't find that location prior to what i found. the more options the better imo.

**Comment by Ryochan7** [comment]
You can grab it from here. And yes, I have not bothered to buy a domain name.

http://23.239.26.40/ds4windows/files/Virtual%20Bus%20Driver.zip

**Comment by deeboshit2992** [comment]
You can grab it from here. And yes, I have not bothered to buy a domain name. "LOL, WE DONT MIND ABOUT THE DOMAIN NAME"
thanks

**Comment by rhx3030** [comment]
go to https://github.com/Ryochan7/DS4Windows/releases
it's a great solution

**Comment by thebitstick** [comment]
wow almost a year and no fix?

**Comment by Belzher** [comment]
Thank you so much Gumija and Ryochan7 for the ScpDriver.exe solution, I was also facing this annoying problem.

**Comment by dirm02** [comment]
I cannot detect the PS4 controller ; What I do ?

**Comment by sweetkid169** [comment]
My suggestion to all is to use input mapper. It's an alternate program since this one is no longer properly supported. Consider the unofficial successor.

**Comment by Ryochan7** [comment]
I test InputMapper fairly frequently but neither version 1.6 nor 1.7 work that well. Neither version can get the most basic DS4 to XInput mapping system right. There have been some interesting developments but Wobbles really should have just focused on getting XInput mapping working well with 1.7 before jumping into implementing other features. Testing the most recent 1.7 build (1.7.6811.34320) with HTML5 Gamepad Tester shows that the full analog stick range is not even being used for the output XInput device.

https://github.com/Ryochan7/DS4Windows

**Comment by sweetkid169** [comment]
Again it's an alternative. Use whatever works for you. This is the option for those that don't want to try and debug this... Our can't get this working... Input mapper is another option. That one is just actively supported. Just updated this month... Not saying one is necessarily better then the other... Everyone can choose for what to use.. But may not know of other options.

**Comment by trakoitas** [comment]
Still an issue
